docs/assistant/cloud/libraries/organize-assets.md

```diff
@@ -0,0 +1,59 @@
+---
+id: organize-assets
+title: Organizing Assets
+sidebar_label: Organizing Assets
+---
+
+Assistant Pro Collections work in a similar way to WordPress post categories.  You can add more than one collection to any creative asset (posts, pages, images …) and that creative asset will show up under each collection.  This way of organizing files is preferred over folders, because files can be cross referenced in multiple places.
```

Comment:
This is the first time I'm seeing the term "Assistant Pro Collections". Define it? Or introduce the concept higher up in the documentation?

Reply:
{ The term should be Library Collection defined on the [Terminology](https://deploy-preview-201--angry-gates-bead98.netlify.app/assistant/#terminology) page. }

Updated Copy: 

Assistant Pro’s Library Collections work in a similar way to WordPress post categories.  You can add more than one collection to any creative asset (posts, pages, images …) and that creative asset will show up under each collection.  This way of organizing files is preferred over folders, because files can be cross referenced in multiple places.